lms/static/js/student_profile/views/share_modal_view.js

```diff
@@ -0,0 +1,48 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'moment',
+            'text!templates/student_profile/share_modal.underscore'],
+        function (gettext, $, _, Backbone, Moment, badgeModalTemplate) {
+
+            var ShareModalView = Backbone.View.extend({
+                attributes: {
+                    'class': 'badges-overlay'
+                },
+                events: {
+                    'click .badges-modal': function (event) {event.stopPropagation();},
+                    'click .badges-modal .close': 'close',
+                    'click .badges-overlay': 'close',
+                    'keydown': 'keyAction',
+                    'focus .focusguard-start': 'focusGuardStart',
+                    'focus .focusguard-end': 'focusGuardEnd'
+                },
+                focusGuardStart: function () {
+                    // Should only be selected directly if shift-tabbing from the start, so grab last item.
+                    this.$el.find("a").last().focus();
+                },
+                focusGuardEnd: function() {
+                    this.$el.find(".badges-modal").focus();
+                },
+                close: function () {
+                    this.$el.fadeOut('short', 'swing', _.bind(this.remove, this));
+                    this.options.shareButton.focus();
+                },
+                keyAction: function (event) {
+                    if (event.keyCode === $.ui.keyCode.ESCAPE) {
+                        this.close();
+                    }
+                },
+                ready: function() {
+                    // Focusing on the modal background directly doesn't work, probably due
+                    // to its positioning.
+                    this.$el.find('.badges-modal').focus();
+                },
+                render: function () {
+                    this.$el.html(_.template(badgeModalTemplate, this.model.toJSON()));
```

Comment:
I don't think this usage is correct. _.template returns something now that must be executed. See https://openedx.atlassian.net/browse/FEDX-66 for details of underscore changes.